zerver/lib/send_email.py

```diff
@@ -39,11 +39,13 @@ def build_email(template_prefix, to_email, from_name=None, from_address=None,
     if from_address is None:
         from_address = FromAddress.NOREPLY
     from_email = formataddr((from_name, from_address))
-    reply_to = None
+
     if reply_to_email is not None:
-        reply_to = [reply_to_email]
+        reply_to = reply_to_email
+    else:
+        reply_to = from_email
 
-    mail = EmailMultiAlternatives(subject, message, from_email, [to_email], reply_to=reply_to)
+    mail = EmailMultiAlternatives(subject, message, from_email, [to_email], reply_to=[reply_to])
```

Comment:
I couldn't think of a situation where that would be useful? In general, extra/unused generality makes code harder to understand and optimize, so I think it is better to keep things specific unless there is a story in which the generality would be used. 

In any case, this function only accepts a single reply_to_email at the moment anyway (the reply_to_email is a Text, rather than a List[Text]).